### ISSUE TITLE
Bitbucket Cloud 503 Service Unabailable

### ISSUE THREAD

**Issue by vseveryn** [mibexsoftware/sonar-bitbucket-plugin#72]
### Actual Behavior

Bitbucket Cloud shows "Add-on Sonar for Bitbucket is not responding. Wait or cancel?"

Seeing 503 Service Unavailable Request during call https://ch-mibex-sonar4bitbucket.herokuapp.com/sonar-dashboard...

### Plug-in version, SonarQube version, CI system, build type

plugin 1.2.3, sonarqube 6.7.1

**Comment by vseveryn** [comment]
found similar issue here https://mibexsoftware.atlassian.net/browse/SONAR4BB-6

**Comment by montrin** [comment]
Hi,

Could you please send me the hyperlink to your repository so that I can investigate the case.

Thanks
Monica

**Comment by vseveryn** [comment]
Hi Monica, it is here https://bitbucket.org/vseveryn/social-network

**Comment by montrin** [comment]
Okay thanks. When I try to access your sonarqube host (got it from your sonar.json file in the repo) I get a timeout error. This is probably the reason why the plugin cannot access it. The host must be online available/ not behind any firewall, otherwise it will not work because the plugin runs in the cloud. Unfortunately I see, we do not log an expressive warning and I’ll change this. 

Are you behind any firewall or does the access on your sonarqube work?

**Comment by vseveryn** [comment]
maybe you got right in time when it was down, it is up now

**Comment by montrin** [comment]
Unfortunately, I still get a timeout.

**Comment by vseveryn** [comment]
are you using this url? https://gyazo.com/58a1f57cefb30ac9e64ae9dc90f3e0cd

**Comment by montrin** [comment]
Yes, I use the same url as you do. The plugin is hosted on Herokus EU region. The IP addresses are dynamic, but the region's IP range should be stable. See here for more information: https://kb.heroku.com/i-need-to-whitelist-heroku-dynos-what-are-ip-address-ranges-in-use-at-heroku

Is it possible to use the domain name of the plugin to whitelist the plugin on your server?

**Comment by vseveryn** [comment]
it seems that the problem is in that we don't have public static ip and the one we use is not available outside ISP network

**Comment by sabrikrdnz** [comment]
Hi vseveryn,

Have you got the solution of this problem? I'm facing this as well.

Thanks,
Sabri
